### PR TITLE
Build Windows (2nd attempt)

### PR DESCRIPTION
**Pull request by jakirkham**
Closes https://github.com/conda-forge/blosc-feedstock/pull/4
Closes https://github.com/conda-forge/blosc-feedstock/pull/6
Closes https://github.com/conda-forge/blosc-feedstock/pull/15

Tries to build on Windows for VS 2008 and VS 2015 in 32-bit and 64-bit varieties. Hopefully overcomes previous shortfallings.

### CODE REVIEW

**Comment by jakirkham** [comment]
Almost all the way on VS 2008 (both 32-bit and 64-bit), but then it has an issue right at the end. Snippet of the error below from [the log]( https://ci.appveyor.com/project/conda-forge/blosc-feedstock/build/1.0.22/job/yrtwkvohqc21pwll#L881 ). Any thoughts @FrancescAlted?

```
Building C object compat/CMakeFiles/filegen.dir/filegen.c.obj
filegen.c
C:\bld\blosc_1520148929217\work\c-blosc-1.14.0\compat\filegen.c(80) : error C2143: syntax error : missing ';' before 'type'
C:\bld\blosc_1520148929217\work\c-blosc-1.14.0\compat\filegen.c(82) : error C2065: 'fsize' : undeclared identifier
C:\bld\blosc_1520148929217\work\c-blosc-1.14.0\compat\filegen.c(82) : error C2065: 'fsize' : undeclared identifier
NMAKE : fatal error U1077: 'C:\PROGRA~2\MICROS~1.0\VC\bin\cl.exe' : return code '0x2'
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,22 +7,32 @@ package:
...
 requirements:
   build:
+    - python       # [win]
     - toolchain
     - cmake
-    - snappy 1.1.6
-    - zlib 1.2.11
-  run:
-    - snappy 1.1.6
-    - zlib 1.2.11
```
Comment (jakirkham) [inline recipe/meta.yaml]: These weren't actually being used as we hadn't set the right build flags to go with them. So Blosc still was using its internal copies of these. Raised issue ( https://github.com/conda-forge/blosc-feedstock/issues/18 ) to unvendor these correctly.

**Comment by jakebolewski** [comment]
awesome, thanks @jakirkham for pushing this through!

**Comment by jakirkham** [comment]
There appears to have been some issue with the install location of the DLLs, which will break things. Had to pull the binaries as a result unfortunately. Have raised issue ( https://github.com/conda-forge/blosc-feedstock/issues/19 ). If anyone has time to submit a PR to fix this, would encourage you to do so.

**Pull request merged**